modules/user_accounts/php/NDB_Form_user_accounts.class.inc

```diff
@@ -737,7 +737,12 @@ class NDB_Form_User_Accounts extends NDB_Form
 
         //------------------------------------------------------------
 
+        // checking if the account can be rejected (no password hash
+        // pending approval)
+        $this->tpl_data['can_reject'] =$this->_canRejectAccount($this->identifier);
```

Comment:
Yes. The button populates on the front end if the account can be reject, and calls an ajax script which checks if the user performing the deletion is admin and if the account can be rejected.